serve/routes.js

```diff
@@ -71,6 +85,30 @@ module.exports = function(app, passport){
 		// call makeDir function here with appropriate function paramters from req
     });
 
+    app.get('/cd', function(req, res) {
+			var path_folder =  './user_data/' + req.user.local.email  + '/' + req.query.directory +'/' + req.query.nameofdir;
+			if(/^[a-zA-Z]/.test(req.query.nameofdir)){
```

Comment:
What is the use of this regex test?

Reply:
I used this regex because if someone only enters ' cd ////// ' then it should outputs what actual terminal does !! To find if any alphabet character is present in foldername , i used it .